utils/ort/src/test/kotlin/OkHttpClientHelperTest.kt

```diff
@@ -58,6 +59,24 @@ class OkHttpClientHelperTest : WordSpec({
 
             clientA shouldBeSameInstanceAs clientB
         }
+
+        "return different clients when specifying different configurations" {
```

Comment:
Would it make sense to say here: different configuration instances with same values?

(To emphasize that it is just about instance equality, not about the equality of the values it sets).

Reply:
> different configuration instances with same values

But that's the test case below, isn't it?

Reply:
The proposal was for the one in line 63 - anyhow, leaving it up to you.

Reply:
> The proposal was for the one in line 63 - anyhow, leaving it up to you.

What I was trying to say is that IMO it's incorrect to rename the test in line 63 to "different configuration instances with same values" as that's a different case that's already tested in line 72.